prime-router/src/main/resources/metadata/hl7_mapping/resources/Device/PRT.yml

```diff
@@ -13,9 +13,51 @@ elements:
     value: [ '%prtDeviceIndex + 1' ]
     hl7Spec: [ '%{hl7OBXFieldPath}-1' ]
```

Comment:
Is `observation-set-id` a mistake? I want to assume it's correct but I can't wrap my head around why we would be setting OBX-1 here.

Reply:
I tried removing this mapping to see what would happen and it looks like it might have only been there to let the EI to Device UDI Carrier integration test to pass. That also happens to be the test I based PRT to Device off of. I suspect the issue is actually due to the technically invalid OBX segment in the test, but I would need more time to wrap my head around it to make sure this didn't have some other obscure use we're depending on.

Reply:
Ticket?